openfermioncirq/gates/four_qubit_gates.py

```diff
@@ -183,13 +188,14 @@ def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs
 
     def __repr__(self):
         if self.exponent == 1:
-            return 'DoubleExcitation'
-        return 'DoubleExcitation**{!r}'.format(self.exponent)
+            return 'ofc.DoubleExcitation'
+        return '(ofc.DoubleExcitation**{})'.format(proper_repr(self.exponent))
```

Comment:
Why the parentheses?

Reply:
If you remove the parens the tests will fail. We require that repr's be "dottable", i.e. that you can just concatenate ".bla()" on and it works. This is not the case for a repr like `a**-1`. The dot hits the -1 instead of the whole expression.

Reply:
:+1: